src/docs/event.yaml

```diff
@@ -0,0 +1,168 @@
+tags:
```

Comment:
This file contains some minor indentation issues. These issues are fixed in this gist: https://gist.github.com/muhammed-shihebi/9d195c5b5a69e27c2f8ad87c6107f730